src/Maker/MakeEntity.php

```diff
@@ -376,6 +382,66 @@ private function askForNextField(ConsoleStyle $io, array $fields, string $entity
             $data['nullable'] = true;
         }
 
+        $data['apiFilter'] = null;
+
+        if (true === $apiOptions) {
+            $availableFilters = [
+                'SearchFilter',
+                'DateFilter',
+                'BooleanFilter',
+                'NumericFilter',
+                'RangeFilter',
+                'ExistsFilter',
+                'OrderFilter',
+                'MatchFilter',
+                'TermFilter',
+            ];
+
+            $apiFilter = null;
+            while (null === $apiFilter) {
+                $question = new Question('Do you want to add a filter for your api resource? (enter <comment>?</comment> to see all filters)', 'No');
+                $question->setAutocompleterValues($availableFilters);
+                $apiFilter = $io->askQuestion($question);
+
+                if ('No' === $apiFilter) {
```

Comment:
Maybe could we just hint the user to type `enter` to skip (as when you're done adding fields).

Reply:
Done.